drbd/res.sls

```diff
@@ -24,8 +24,10 @@
         fixed_rate: {{ res.fixed_rate|default(True) }}
         resync_rate: '{{ res.resync_rate|default("100M") }}'
         c_plan_ahead: {{ res.c_plan_ahead|default(20) }}
-        c_max_rate: '{{ res.c_max_rate|default("100M") }}'
+        c_delay_target: '{{ res.c_delay_target|default(1) }}'
         c_fill_target: '{{ res.c_fill_target|default("10M") }}'
+        c_max_rate: '{{ res.c_max_rate|default("100M") }}'
```

Comment:
More default values here. Shouldn't they go in the `defaults` file?

Reply:
I don't find a better solution for this. 
Since the default `res` section is empty in `defaults.yaml`. So only have default values like `c_max_rate` here